docs/pages/access-controls/access-requests/role-requests.mdx

```diff
@@ -56,138 +48,146 @@ metadata:
   name: dba
 spec:
   allow:
-    # ...
-  deny:
-    # ...
+    db_labels:
+      '*': '*'
   options:
-    # ...
     # Only allows the contractor to use this role for 1 hour from time of request.
     max_session_ttl: 1h
 ```
 
-**Admin Role**
-This role allows the admin to approve the contractor's request.
+**Approver Role**
+
+This role allows users to approve requests for the `dba` role.
 
 ```yaml
 kind: role
 version: v5
 metadata:
-  name: admin
+  name: approver
 spec:
   allow:
     # `review_requests` permits the listed roles to be approved
     review_requests:
       roles:
       - 'dba'
-    rules:
-    # `access_request` is part of Access Workflows introduced in 4.2
-    # `access_request` should only be given to Teleport Admins.
-    # For a full list of allow-rules, see
-    # https://goteleport.com/docs/access-controls/reference/
-    - resources:
-      - access_request
-      verbs:
-      - list
-      - read
-      - update
-      - delete
-  deny:
-    # ...
-  options:
-    # ...
 ```
 
-```code
-$ tsh login --request-roles=dba teleport-cluster
-# Seeking request approval... (id: bc8ca931-fec9-4b15-9a6f-20c13c5641a9)
-```
+<Admonition type="warning" title="Requires Teleport Enterprise">
+Roles containing a `review_requests` rule can only be used in Teleport
+Enterprise. In OSS Teleport, Access Requests must be approved by an admin
+running `tctl`.
+</Admonition>
+
+## Requesting Access
+
+While Teleport Enterprise supports the same CLI-based workflows for requesting
+access to roles, most users will prefer to request access via the web UI.
+
+To request access to one or more roles, navigate to the access requests page.
+You can find this page by selecting *Resources* on the side bar, expanding the
+*Access Requests* menu, and selecting *New Request*.
+
+From here, the dropdown will allow you to request access to roles or to specific
+resources. Select roles in this dropdown and you will be presented with a list
+of roles you are permitted to request access to.
+
+Click *ADD TO REQUEST* to include a role in the pending request. When all desired
```

Comment:
```suggestion
Click **ADD TO REQUEST** to include a role in the pending request. When all desired
```

Reply:
Not sure why this one is showing as outdated, but it's relevant to the current version.